README.org

```diff
@@ -1,17 +1,33 @@
+* Eventually Matrix
+
+  [[https://en.wikipedia.org/wiki/Eventual_consistency][Eventually]] Matrix is a
+  repository of knowledge about the Matrix ecosystem maintained by the community
+  for the community. It is the "bleeding edge" counterpart of the official
+  Matrix.org repositories in terms of documentation, SDK and libraries.
+
 * Purpose                                              :matrix:eventually_matrix:
 
-  Eventually Matrix is a repository of knowledge about the Matrix ecosystem maintained
-  by the community for the community. In broad strokes, this repository exists for
-  the following purposes in roughly the order they're listed.
+  In broad strokes, this repository exists for the following purposes in roughly the
+  order they're listed.
 
+  - Allow Matrix community to easily document the current behavior of implementations
```

Comment:
Should this be split into a separate repository from our more speculative efforts? Not sure at all.

Reply:
We can when there is actually someone doing speculative work. So far it's only me documenting what's existing out there. I wouldn't think too far ahead here... let's create as the need arise, let's just be on the ball about it.

Reply:
I think it's very important to keep the two separate. IMO, it's a documentation vs. specification kind of deal: one documents the behavior that is seen, the other specifies the behavior that ought to exist.

Reply:
@Magnap do you see this blocking the README merge, as it's related to something outside this repo?

Reply:
If the section about what each repo is for is removed, then I don't think it does. If we keep that section in, we should do so with a clearer idea of the intended workflow. I prefer the former.